### DISCUSSION TITLE
How to Suppress Excessive Logs in Console After Switching to GetMaterialApp?

### DISCUSSION THREAD

**Discussion by khushpajwani** [jonataslaw/getx#3152] [Q&A] [answered]
After switching from `MaterialApp` to `GetMaterialApp`, my console is now flooded with numerous logs that I don't need. Does anyone know how to disable or suppress these logs?

**Comment by dharambudh1** [comment, marked as answer]
Yes, you can disable the logs by setting the `enableLog` argument to `false` in `GetMaterialApp`.

Example: 

```
GetMaterialApp(
      enableLog: false,
```

**Comment by khushpajwani** [comment]
Okay, thank you!
